azurelinuxagent/ga/update.py

```diff
@@ -362,20 +362,21 @@ def run(self, debug=False):  # pylint: disable=R0912
                     if last_etag != exthandlers_handler.last_etag:
                         self._ensure_readonly_files()
                         duration = elapsed_milliseconds(utc_start)
-                        logger.info('ProcessGoalState completed [incarnation {0}; {1} ms]',
-                                    exthandlers_handler.last_etag,
-                                    duration)
+                        activity_id, correlation_id = exthandlers_handler.get_activity_and_correlation_id()
+                        msg = 'ProcessGoalState completed [Incarnation: {0}; {1} ms; Activity Id: {2}; Correlation Id: {3}]'.format(
```

Comment:
Let's also log it when we fetch the goal state (you can remove it from here if you don't want to log it twice)

Reply:
I already have that - 

>logger.info(
                    "ProcessGoalState started [Incarnation: {0}; Activity Id: {1}; Correlation Id: {2}]".format(etag,
                                                                                                                activity_id,
                                                                                                                correlation_id))

Is that not what you mean?

Reply:
cool, i missed that one